thriftbp/server.go

```diff
@@ -0,0 +1,41 @@
+package thriftbp
+
+import (
+	"time"
+
+	"github.com/apache/thrift/lib/go/thrift"
+
+	"github.com/reddit/baseplate.go/log"
+)
+
+// ServerConfig is the arg struct for NewThriftServer.
+type ServerConfig struct {
+	// The endpoint address of your thrift service
+	Addr string
+
+	// The timeout for the underlying thrift.TServerSocket transport.
+	Timeout time.Duration
+
+	// A log wrapper that is passed to the "Wrap" call and server.SetLogger
```

Comment:
After you removed logger from Wrap function, this comment should be changed to clarify that it's only used by `thrift.SimpleServer`.